source-browser.html

```diff
@@ -181,13 +209,51 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(item) {
+          $('#li').append(
+              $('<a>', {
+                  id: 'li' + item,
```

Comment:
Use string interpolation instead of concatenation i.e. 
    
    `li${item}`

Also, `id`s should be id-agnostic if possible. So, something like `${item}-nav-link` perhaps?